cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -255,19 +256,18 @@ func setupSignalChannel(cancel context.CancelFunc) {
 }
 
 func newAPIServer(api api.ProxyAPI) *http.Server {
-	c := cors.New(cors.Options{
-		AllowCredentials: true,
-	})
-	srv := &http.Server{
+	return &http.Server{
 		Addr:    apiRouterPort,
-		Handler: c.Handler(router.CreateAPIRouter(api)),
+		Handler: router.CreateAPIRouter(api),
 	}
-	return srv
 }
 
-func newJenkinsAPIServer(jenkinsAPI jenkinsapi.JenkinsAPI) *http.Server {
+func newJenkinsAPIServer(jenkinsAPI jenkinsapi.JenkinsAPI, config configuration.Configuration) *http.Server {
+	allowedMethods := strings.Split(config.GetAllowedOrigins(), ", ")
```

Comment:
rather let the config.GetAllowedOrigins return []string{} so that the caller need not worry about the delimiter. also using a single char `,` as delimiter will be a better choice.

Reply:
Another thing that is that the var name shouldn't be `allowedMethods` :)

Reply:
Ack